### DISCUSSION TITLE
不能在vue3里使用啊？

### DISCUSSION THREAD

**Discussion by elarbor** [magicuidesign/magicui#379] [General]
![image](https://github.com/user-attachments/assets/c97b33de-8b5c-4ed3-b1d5-9c09fb91e001)

**Comment by Yeom-JinHo** [comment]
@elarbor 
At the moment, Magic UI is built specifically for React and the shadcn-based ecosystem, so Vue 3 isn’t officially supported.

Since the components rely on React and related libraries (like motion/react), they won’t work out of the box in a Vue environment.

If you’re using Vue 3, you could still take inspiration from the designs and reimplement the logic in Vue — but there isn’t native support right now.

@dillionverma  If there are any future plans to support Vue 3, it would be great to hear about them.
